xrootd/authorization_test.go

```diff
@@ -510,7 +509,6 @@ func TestMergeConfig(t *testing.T) {
 			err := os.WriteFile(scitokensConfigFile, []byte(configInput), fs.FileMode(0600))
 			require.NoError(t, err)
 
-			config.InitConfigDir(viper.GetViper())
```

Comment:
Sorry for more goose chases -- it doesn't look like the this unit test isolates itself. Can you configure a temp dir somewhere by the other viper config up top?